zulipterminal/ui_tools/views.py

```diff
@@ -122,11 +123,27 @@ def keypress(self, size: Tuple[int, int], key: str) -> str:
         key = super(MessageView, self).keypress(size, key)
         return key
 
+    def update_current_footer(self, message_view: Any) -> None:
+        if hasattr(self.model.controller, 'view'):
```

Comment:
It would be nice to just return if the `view is None` and do the footer computation otherwise.